test/onnx/test_operators.py

```diff
@@ -593,7 +593,7 @@ def test_batchnorm_noaffine(self):
         self.assertONNX(nn.BatchNorm2d(128, affine=False, momentum=0.3), x,
                         keep_initializers_as_inputs=True)
 
-    @skipIfCaffe2
+    @skipIfNoCaffe2
```

Comment:
I think tests in this file should test the non-Caffe2 specific behavior whenever possible, as that's what will affect most users going forard. So I think the old decorator was more appropriate. WDYT?

Reply:
The test was originally added for Caffe2, so I kept this behavior.
I am OK with converting to aten-non-caffe2 or even have both caffe2 and non-caffe2 versions

Reply:
I don't understand how the test could originally be for caffe2 if it was decorated with `skipIfCaffe2`?
Please explain.

Reply:
I did that in the past https://github.com/pytorch/pytorch/commit/9bbe1d632e6536934550b6973366e2acdd4e1995 and now plan to restore original behavior and guard legacy Caffe2 from our changes

Reply:
My vote is to either delete it or keep it as a non-Caffe2 test.

Reply:
ok, changing to non-caffe2 test